src/protobuf-net.Test/Nullables/CollectionsWithNullsTests_SupportNull.cs

```diff
@@ -0,0 +1,79 @@
+﻿using ProtoBuf.Meta;
+using ProtoBuf.Test.Nullables.Abstractions;
+using System.Collections.Generic;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ProtoBuf.Test.Nullables
+{
+    public class CollectionsWithNullsTests_SupportNull : CollectionsWithNullsTestsBase
+    {
+        // 2. if model is tweaked with SupportNull:
+        //  2a. schema has new wrapper layer, "message Foo { repeated NullWrappedBar Items = 1; }" 
+        //      with "group Bar value = 1" **invalid** syntax
+        //  2b. payload has the extra layer with "group"
+        //  2c. null works correctly!
+
+        public CollectionsWithNullsTests_SupportNull(ITestOutputHelper log) 
+            : base(log)
+        {
+        }
+
+        protected override void Setup()
+        {
+            RuntimeTypeModel.Default[typeof(SupportsNullListModel)][1].SupportNull = true;
```

Comment:
note: if we want to test with+without, I would suggest using a bespoke model each time:

```
var model = RuntimeTypeModel.Create();
```

Note that you can't use `Serializer.Whatever(...)`, since `Serialize.*` is a proxy to `RuntimeTypeModel.Default.*`, but: `model.*` should work fine

This avoids tests interfering with eachother - concurrency, order, etc

Reply:
Not to intercept with other test classes, using separate instance for each test class